src/lpython/python_evaluator.cpp

```diff
@@ -319,4 +360,105 @@ Result<std::unique_ptr<LLVMModule>> PythonCompiler::get_llvm3(
 #endif
 }
 
+void print_type(ASR::ttype_t *t, void *data, std::string &result);
+
+std::string PythonCompiler::string_aggregate_type(const struct EvalResult &r) {
```

Comment:
Is this function converting a given aggregate type to string? If yes, it should be called `aggregate_type_to_string()` or `cnvrt_aggregate_type_to_string()` or something similar. `string_aggregate_type` doesn't give any good idea of what the function does and seems rather misleading.